string sensor_name
string command
---
string message
string command_message
